src/components/QuranReader/ReadingView/ReadingViewSkeleton.module.scss

```diff
@@ -0,0 +1,17 @@
+@use "src/styles/breakpoints";
```

Comment:
reuse styles from the actual component

Reply:
we only have one class here `.skeleton`. It's a bit hard to reuse the styling here since
1)  we dont use the actualy verse component
2) the verse component has a lot of whitespace. It will increase the skeleton size. bad perceived size.
![image](https://user-images.githubusercontent.com/12745166/148205878-563f1d3f-ff9e-4fce-9eda-68e37320366d.png)